libs/designer-ui/src/lib/settings/settings.less

```diff
@@ -89,7 +89,6 @@
 
     .msla-setting-section-setting {
       margin-left: 46px;
-      display: flex;
```

Comment:
I think this is so commonly used can you check if recurrence parameters [since they are in different section] in a trigger are rendered correctly and the settings panel as well?

Reply:
this was actually something I did to fix how the recurrence parameters looked. It looks fine in the settings panel